Update acmeta.doc to 2.0

TimRyan99 opened the pull request:
*Version 2.0. 17^th^ Feburary 2020*

Major revision. Added 'platform' category to allow long list of platform classes to be supported. These class types are defined by ICES controlled vocabulary. Mooring category attributes have been removed and can now be specified using the set of platform attributes.

TimRyan99 committed:
Update acmeta.doc to 2.0

*Version 2.0. 17^th^ Feburary 2020*

Major revision. Added 'platform' category to allow long list of platform classes to be supported. These class types are defined by ICES controlled vocabulary. Mooring category attributes have been removed and can now be specified using the set of platform attributes.

gavinmacaulay commented on AcMeta/acmeta.adoc:
```diff
@@ -97,7 +135,7 @@ Cruise metadata:: Attributes that describe the cruise from which the acoustic da
 Ship metadata:: Attributes that describe the ship from which acoustic data were collected. Metadata should provide information that uniquely identifies the ship and its basic specifications to enable an understanding of the type of ship and its purpose.
 
-Mooring metadata:: Attributes that describe the mooring from which acoustic data were collected.
+Platfomg metadata:: Attributes that describe the platform from which acoustic data were collected.
```
```suggestion
Platform metadata:: Attributes that describe the platform from which acoustic data were collected.
```

gavinmacaulay commented on AcMeta/acmeta.adoc:
```diff
@@ -1,5 +1,42 @@
 = A metadata convention for processed acoustic data from active acoustic systems
```
The table of contents and title are generated automatically when asciidoctor generates the pdf or html files, so it is not necessary add them manually.

TimRyan99 committed:
testing erins git instructions with minor edits

TimRyan99 committed:
testing erins instructions

TimRyan99 committed:
Update acmeta.adoc

Took at my manual table of contents following Gavin's advising that these are automatically generated.

TimRyan99 committed:
Update AcMeta/acmeta.adoc

fixed type

Co-Authored-By: Gavin Macaulay <gavin@macaulay.co.nz>
